staff/pedro-garriganogues/gamma-template/index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Page Title</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+    <script src="main.js"></script>
+</head>
+
+<body>
+    <header class="header">
+        <h1 class="green-font">Gamma</h1>
+        <img class="burger" src="./images/bars-solid.svg" alt="404">
+        <section class="redirect">
+            <a class="link" href="index.html">Home</a>
+            <a class="link" href="about.html">About</a>
+            <a class="link" href="contact.html">Contact</a>
+        </section>
+    </header>
+    <section class="black-top">
+        <div>
+        <div class="title">
+            <h1 class="white-font">Capture <span class="green-font">Life</span></h1>
+        </div>
+        <p class="white-font">A fun way to capture & share the moments you life for</p>
+        <div class="gallery-store">
+            <img class="store-link" src="./images/appstore.png" alt="404">
+            <img class="store-link" src="./images/googleplay.png" alt="404">
+        </div>
+        </div>
+        <div>
+            <img class="godhand" src="./images/0.jpg" alt="404">
+        </div>
+    </section>
+
+    <section class="tresamigos">
+        <section class="amigo">
+            <img class="picture" src="./images/2.png" alt="404">
+            <div class="description">
+                <h1>scotcher</h1>
+                <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Voluptatem rem cupiditate doloremque sit
+                    fugit
+                    expedita omnis minus recusandae, nihil magni est, libero sequi nobis in, laboriosam itaque minima
+                    nemo
+                    dolorum?</p>
+            </div>
+            <a class="button-more">Read more</a>
```

Comment:
anchors need href attribute. If you don't have a link you write # in the href attribute.